resources-guide/git.md

```diff
@@ -18,8 +18,12 @@ Linked below are several excellent detailed descriptions and tutorials on the bu
     * Also see the main [Lessons page](https://software-carpentry.org/lessons/) at Software Carpentry for many other tutorials
 * [Happy Git and GitHub for the useR](https://happygitwithr.com/) from Jenny Bryan (for use with RStudio)
 
+### Gitignore
+* [Gitignore generator](gitignore.io) - a website to generate .gitignore files for your project from templates based on language, OS, project framework, etc. If you provide multiple templates, it will generate a single .gitignore file that combines them.
```

Comment:
```suggestion
* [Gitignore generator](https://gitignore.io) - a website to generate .gitignore files for your project from templates based on language, OS, project framework, etc. If you provide multiple templates, it will generate a single .gitignore file that combines them.
```